ocs_ci/utility/utils.py

```diff
@@ -3350,6 +3351,37 @@ def clone_notify():
     return notify_path
 
 
+def get_encryption_kmsid():
```

Comment:
Can we move this one to ocs_ci/utility/kms.py  ?  This is KMS relarted so would prefere move it out of general utility module. This could also fix the circular imports issue.

Reply:
ack

Reply:
done